core-bundle/src/Command/SymlinksCommand.php

```diff
@@ -110,8 +111,11 @@ private function generateSymlinks(): void
         $this->symlink($this->getRelativePath($this->getContainer()->getParameter('kernel.logs_dir')), 'system/logs');
 
         // Symlink the TCPDF config file
-        $this->symlink('vendor/contao/core-bundle/src/Resources/contao/config/tcpdf.php', 'system/config/tcpdf.php');
-    }
+        if (isset($packages['contao/contao'])) {
+            $this->symlink('vendor/contao/contao/core-bundle/src/Resources/contao/config/tcpdf.php', 'system/config/tcpdf.php');
+        } else {
+            $this->symlink('vendor/contao/core-bundle/src/Resources/contao/config/tcpdf.php', 'system/config/tcpdf.php');
+        }    }
```

Comment:
You could use `kernel.bundles` and get the path of the `ContaoCoreBundle`. So you could get rid of the if statement, couldn't you?